packages/jest-leak-detector/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "build"
+  }
```

Comment:
This needs a project reference to `pretty-format`. See https://github.com/facebook/jest/pull/7824/files#diff-0567011fbdfa641e4e705d6820a3d93c as an example

Reply:
Hey! Thanks for your help, updated it.